app/build.gradle

```diff
@@ -92,7 +92,7 @@ dependencies {
     implementation "androidx.lifecycle:lifecycle-livedata-ktx:2.8.6"
     implementation "androidx.lifecycle:lifecycle-viewmodel-compose:2.8.6"
     // TODO: Needs migration to type-safe navigation in v2.8.x
```

Comment:
We can safely remove this comment now :)

Reply:
done